{"uid":"63ddf2b24f8d2db3","name":"CliKintoneTest-103 Should return the error message when exporting records with --fields specified, including fields within a table.","fullName":"features/export.feature#CliKintoneTest-103 Should return the error message when exporting records with --fields specified, including fields within a table.","historyId":"f2dca44267e386ecc681c6deb35ad2dd:d41d8cd98f00b204e9800998ecf8427e","time":{"start":1711697507638,"stop":1711697508742,"duration":1104},"description":"","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"","status":"passed","steps":[{"name":"Given Load app ID of the app \"app_for_export_table\" as env var: \"APP_ID\"","time":{"start":1711697507638,"stop":1711697507639,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And Load app token of the app \"app_for_export_table\" with exact permissions \"view\" as env var: \"API_TOKEN\"","time":{"start":1711697507639,"stop":1711697507639,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When I run the command with args \"record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN --fields Text\"","time":{"start":1711697507639,"stop":1711697508741,"duration":1102},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then I should get the exit code is non-zero","time":{"start":1711697508741,"stop":1711697508741,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And The output error message should match with the pattern: \"Error: The field in a Table cannot be specified to the fields option \\(\\\"Text\\\"\\)\"","time":{"start":1711697508741,"stop":1711697508741,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":5,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"host","value":"fv-az1240-929"},{"name":"language","value":"javascript"},{"name":"framework","value":"cucumberjs"},{"name":"thread","value":"2081"},{"name":"feature","value":"cli-kintone export command"},{"name":"suite","value":"CliKintoneTest-103 Should return the error message when exporting records with --fields specified, including fields within a table."},{"name":"tag","value":"@isolated"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@isolated"]},"source":"63ddf2b24f8d2db3.json","parameterValues":[]}